### PR TITLE
Add ability to override client_properties in Connection

### PR DESCRIPTION
**Pull request by tkram01**
It would be useful to be able to override the client_properties that is sent when the Connection is made. This pull request adds an optional argument to Connection and UriConnection where you can pass in a dict. The default client properties is updated with this dict to come up with the client properties to send to RabbitMQ.

### CODE REVIEW

File: amqpstorm/uri_connection.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ class UriConnection(Connection):
     """
     __slots__ = []
 
-    def __init__(self, uri, lazy=False):
+    def __init__(self, uri, lazy=False, client_properties=None):
```
Comment (eandersson) [inline amqpstorm/uri_connection.py]: Lets put client_properties before lazy.

Comment (tkram01) [reply amqpstorm/uri_connection.py]: All set

**Comment by eandersson** [comment]
This patch looks good in general - if you fix the conflicts I'll merge it.

**Comment by tkram01** [comment]
Any chance we can revive this? I would still find this really useful. Do you want me to make a clean patch?

**Comment by eandersson** [comment]
Sure - I can look at it this weekend.

**Pull request closed without merging**